azure-pipelines/build-release.yml

```diff
@@ -26,50 +26,77 @@ stages:
         strategy:
           maxParallel: 10
           matrix:
+            Linting_Py_3_6:
```

Comment:
Commit "ci: Use tox instead of pipenv for analyse and test" doesn't do anything with pipenv. Should it be called instead "ci: Use tox to run linting and pytest"?